digits/views.py

```diff
@@ -247,9 +248,13 @@ def path_autocomplete():
 
     """
     path = flask.request.args.get('query','')
+    suggestions = glob.glob(path+"*")
+    if os.name == 'nt':
```

Comment:
Can we use `if platform.system() == 'Windows'` here as we do elsewhere in the code?